src/api/messages.js

```diff
@@ -12,6 +12,11 @@ const messages = defineMessages({
         description: 'Error message when an app activity deletion fails',
         defaultMessage: 'There was an error deleting this item.',
     },
+    annotationUpdateErrorMessage: {
```

Comment:
It doesn't look like comments actually has this UX -- two comments: (1) should this message be `commentUpdateErrorMessage` since we're treating these annotations as comments and (2) should we update the existing update comment flow to behave the same as our annotation flow like we do for `deleteAnnotation`?

Reply:
Yes to both comments